blank_project/src/index.html

```diff
@@ -76,7 +76,7 @@
           <button id="sign-out-button" >Sign-out</button>
       </div> 
   </div>
-  <script src="https://cdn.jsdelivr.net/gh/nearprotocol/nearlib@0.19.1/dist/nearlib.js"></script>
+  <script src="https://cdn.jsdelivr.net/gh/nearprotocol/nearlib@0.21.0/dist/nearlib.js"></script>
```

Comment:
does dependabot do these types of deps? I guess not but it'd be nice to have something like that

Reply:
I'll update all templates to use Parcel bundler (similar to wallet and near-chess GitPod demo). We won't have this kind of dependencies anymore when we deprecate studio.